entity-management-web/src/main/java/eu/europeana/entitymanagement/web/service/ScoringService.java

```diff
@@ -193,30 +160,12 @@ public EntityMetrics getMaxOverallMetrics() throws IOException {
 	return maxOverallMetrics;	
     }
 
-    private Integer getEnrichmentCount(Entity entity) {
-	String queryStr = emConfiguration.getEnrichmentsQuery();
-	queryStr = String.format(queryStr, entity.getEntityId());
-	SolrQuery query = new SolrQuery(queryStr);
-	return getCountBySolrQuery(query, entity.getEntityId());
-    }
-
-    private Integer getCountBySolrQuery(SolrQuery query, String entityId) {
-	query.setRows(0);
-
-	try {
-		Instant start = Instant.now();
-	    QueryResponse rsp = searchApiSolrClient.query(query, METHOD.POST);
+	private Integer getEnrichmentCount(Entity entity) {
+    	boolean isOrg = entity.getType().equals(EntityTypes.Organization.getEntityType());
 
-		if (logger.isDebugEnabled()) {
-			logger.debug("Retrieved enrichmentCount for entityId={} in {}ms", entityId,
-					Duration.between(start, Instant.now()).toMillis());
-		}
-
-	    return (int) rsp.getResults().getNumFound();
-	} catch (Exception e) {
-	    throw new ScoringComputationException("Unexpected exception occured when retrieving pagerank: " + entityId, e);
+		return enrichmentCountQueryService.getEnrichmentCount(entity.getEntityId(), isOrg);
```

Comment:
it is likely that we will need the entity type as parameter, to identify the corect solr field for computing enrichment count